frontend/components/dialogs/ProjectHistoryDialog.vue

```diff
@@ -491,6 +526,15 @@ export default {
       }
       return ''
     },
+    parseCustomAnswers(answers) {
```

Comment:
does this work for both donor custom and country custom answers?

Reply:
No, `donor_answers` and `country_answers` are ignored (just like in export/import functions)